ydb/core/tx/schemeshard/ut_restore/ut_restore.cpp

```diff
@@ -49,6 +50,18 @@ namespace {
         Aws::ShutdownAPI(Options);
     }
 
+    std::string FindAuditLine(const std::vector<std::string>& auditLines, const std::string& substr) {
```

Comment:
There is same function in `ut_export.cpp`. May be move it to `ut_helpers/auditlog_helpers.*`?

Reply:
Certainly. Thank you for spotting this.